### DISCUSSION TITLE
Bot clean up items from crates in lost cargo mission, failing it

### DISCUSSION THREAD

**Discussion by SomeRandomNoobKekeke** [FakeFishGames/Barotrauma#13928] [Bug reports]
### Disclaimers  

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Ofc you can tell bots to ignore the crates, but still could be a bit confusing why mission is failed, considering that all cargo is technically on board
And it's potentially too easy to grief this mission by just drinking one ethanol from the stack

### Reproduction steps

Start any lost cargo mission
Bring crates on board
Order some bot to clean up items
Watch him clean up items from the crates
Dock, see that mission is failed

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/pull/5344/commits/dfc93579783883abd5fe0d656cf533fa385a20c8

**Comment by Jasontti** [comment]
Tested the fix and found no issues.
Closing discussion as resolved.
